controllers/login.js

```diff
@@ -2,57 +2,64 @@
  * One Controller per layout view
  */
 
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const { api } = require('../api');
-const authHelper = require('../helpers/authentication');
+const { api } = require("../api");
+const authHelper = require("../helpers/authentication");
 
 // Login
 
-router.post('/login/', function (req, res, next) {
-    const username = req.body.email; // TODO: sanitize
-    const password = req.body.password; // TODO: sanitize
-
-    const login = (data) => {
-        return api(req).post('/authentication', {json: data}).then(data => {
-            res.cookie('jwt', data.accessToken, {expires: new Date(Date.now() + 2 * 60 * 60 * 1000)});
-            res.redirect('/login/success/');
-        }).catch(_ => {
-            res.locals.notification = {
-                'type': 'danger',
-                'message': 'Login fehlgeschlagen.'
-            };
-            next();
-        });
-    };
+router.post("/login/", function (req, res, next) {
+  const username = req.body.email; // TODO: sanitize
+  const password = req.body.password; // TODO: sanitize
 
-    return login({strategy: 'local', username, password});
+  const login = (data) => {
+    return api(req)
+      .post("/authentication", { json: data })
+      .then((data) => {
+        res.cookie("jwt", data.accessToken, {
+          expires: new Date(Date.now() + 24 * 60 * 60 * 1000),
```

Comment:
Looks like this is the only relevant change? The others look like re-formatted code, right?